src/lib/mongoBackend/MongoCommonRegister.cpp

```diff
@@ -82,7 +83,7 @@ static bool processSubscriptions
                                          trigs->attrL,
                                          mapSubId,
                                          trigs->reference,
-                                         trigs->format,
+                                         JSON,
```

Comment:
We don't need a new issue. You can use comments on #2015 to keep track of pending things related with notifications NGSIv2 functionality.